db/model/Gdoc/rawToEnriched.ts

```diff
@@ -829,7 +835,12 @@ function parseStickyRight(
         parseErrors: [error],
     })
     const { left, right } = raw.value
-    if (!left.length || !right.length) {
+    if (
+        left === undefined ||
+        right === undefined ||
+        !left.length ||
+        !right.length
+    ) {
```

Comment:
`!left?.length` ? Or is there the chance to get `left === null` and you want to throw in that case?